models.py

```diff
@@ -1,13 +1,81 @@
-from sqlalchemy import Column, Integer, String, Float
+import enum
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Float,
+    Table,
+    ForeignKey,
+    Enum,
+    DateTime,
+    func,
+)
+from sqlalchemy.orm import relationship
 
 from database import Base
 
 
+ACCESS_LEVELS = {
+    "superadmin": 3,
+    "admin": 2,
+    "regular": 1,
+}
+
+
+class AccountType(str, enum.Enum):
+    Regular = "regular"
+    Admin = "admin"
+    Superadmin = "superadmin"
+
+
+exercise_category = Table(
+    "exercise_category",
+    Base.metadata,
+    Column("category_name", String, ForeignKey("category.category")),
+    Column("exercise_id", Integer, ForeignKey("exercise.id")),
+)
+
+
+class Complexity(enum.Enum):
+    _easy = "easy"
+    _medium = "medium"
+    hard = "hard"
+
+
 class Exercise(Base):
     __tablename__ = "exercise"
 
     id = Column(Integer, primary_key=True, index=True, autoincrement=True)
     title = Column(String)
+    complexity = Column(Enum(Complexity), nullable=True)
     text = Column(String)
-    category = Column(String)
-    complexity = Column(Float)
+    category = relationship(
+        "Category", secondary=exercise_category, back_populates="exercises"
+    )
+    date = Column(DateTime, default=func.now(), onupdate=func.now())
+
+
+class Account(Base):
+    __tablename__ = "account"
+
+    id_account = Column(Integer, primary_key=True, index=True, autoincrement=True)
+    email = Column(String, unique=True)
+    password = Column(String)
+    account_category = Column(Enum(AccountType))
+
+
+class User(Base):
+    __tablename__ = "user"
+
+    id_user = Column(Integer, primary_key=True, index=True, autoincrement=True)
+    id_account = Column(Integer)
```

Comment:
Should reference account(id_account)
id_account = Column(Integer, ForeignKey('account.id_account'))

 account = relationship("Account", back_populates="users")